config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -1,401 +1,4 @@
-    fork-per-release: "true"
-    fork-per-release-periodic-interval: 1h 2h 6h 24h
-    testgrid-dashboards: sig-release-master-blocking, sig-node-release-blocking
-    testgrid-tab-name: node-kubelet-master
-    testgrid-alert-email: kubernetes-sig-node+testgrid@googlegroups.com,kubernetes-sig-node-test-failures@googlegroups.com
-    description: "Uses kubetest to run a subset of node-e2e tests (+NodeConformance, -Flaky|Serial)"
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-  spec:
-    containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211210-0c6ec8feca-master
-      args:
-      - --repo=k8s.io/kubernetes=master
-      - --timeout=90
-      - --root=/go/src
-      - --scenario=kubernetes_e2e
-      - --
-      - --deployment=node
-      - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml
-      - --node-test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
-      - --node-tests=true
-      - --provider=gce
-      - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Serial\]"
-      - --timeout=65m
-      env:
-      - name: GOPATH
-        value: /go
-      resources:
-        limits:
-          cpu: 6
-          memory: 6Gi
-        requests:
-          cpu: 6
-          memory: 6Gi
-- name: ci-kubernetes-node-kubelet-kubetest2
-  cluster: k8s-infra-prow-build
-  interval: 1h
-  decorate: true
-  extra_refs:
-  - org: kubernetes
-    repo: kubernetes
-    base_ref: master
-    path_alias: k8s.io/kubernetes
-  - org: kubernetes
-    repo: test-infra
-    base_ref: master
-    path_alias: k8s.io/test-infra
-  decoration_config:
-    timeout: 65m
-  annotations:
-    fork-per-release: "true"
-    fork-per-release-periodic-interval: 1h 2h 6h 24h
-    testgrid-dashboards: sig-node-critical
-    testgrid-tab-name: node-kubelet-master-kubetest2
-    testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
-    description: "Uses kubetest2 to run a subset of node-e2e tests (+NodeConformance, -Flaky|Serial)"
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-  spec:
-    containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211210-0c6ec8feca-experimental
-      resources:
-        limits:
-          cpu: 6
-          memory: 6Gi
-        requests:
-          cpu: 6
-          memory: 6Gi
-      env:
-      - name: GOPATH
-        value: /go
-      command:
-      - runner.sh
-      args:
-      - kubetest2
-      - noop
-      - --test=node
-      - --
-      - --repo-root=.
-      - --gcp-zone=us-west1-b
-      - --parallelism=8
-      - --focus-regex=\[NodeConformance\]
-      - --skip-regex=\[Flaky\]|\[Serial\]
-      - --test-args=--kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
-      - --image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/image-config.yaml
-
-- name: ci-kubernetes-node-kubelet-alpha
```

Comment:
is the https://github.com/kubernetes/test-infra/blob/1722302a6619af14f5694c1968d362a3a32de650/config/jobs/kubernetes/sig-node/containerd.yaml#L457 containerd match for it?

Reply:
From 26adeb7 commit message:

>    Remove ci-kubernetes-node-kubelet-alpha/-kubetest2
>    
>    Covered by ci-cri-containerd-e2e-cos-gce-alpha-features and new cri-o
>    job ci-crio-cgroupv1-node-e2e-alpha.